stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -41,12 +41,9 @@ spec:
       {{- if .Values.rbacEnabled}}
       serviceAccountName: {{ template "rabbitmq.fullname" . }}
       {{- end }}
-      affinity:
-{{- if .Values.affinity }}
-    {{- with .Values.affinity }}
-{{ tpl . $ | indent 8 }}
-    {{- end }}
-{{- end }}
+      {{- if .Values.affinity }}
+      affinity: {{- include "rabbitmq.tplValue" (dict "value" .Values.affinity "context" $) | nindent 8 }}
```

Comment:
Should this be current context (`.`) instead of root context (`$`)?

Reply:
Yes. It should be (`.`)

Reply:
Fixed.